nc-games-fe/src/Components/utils.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+const gamesApi = axios.create({
+    baseURL: `https://nc-games-portfolio-api.onrender.com/api`
+})
```

Comment:
Nice to see you have a base URL set up here. It makes it cleaner and easier to read.